llvm/test/MC/AMDGPU/gfx1250_asm_vopd.s

```diff
@@ -9937,3 +9937,6339 @@ v_dual_mul_f32 v255, -1, v4 :: v_dual_subrev_f32 v6, src_scc, v5
 v_dual_mul_f32 v6, null, v5 :: v_dual_subrev_f32 v255, 0xaf123456, v4
 // GFX12: v_dual_mul_f32 v6, null, v5 :: v_dual_subrev_f32 v255, 0xaf123456, v4 ; encoding: [0x7c,0x0a,0xcc,0xc8,0xff,0x08,0xfe,0x06,0x56,0x34,0x12,0xaf]
 // W64-ERR: :[[@LINE-2]]:{{[0-9]+}}: error
+
+v_dual_sub_f32 v255, v4, v2 :: v_dual_add_f32 v6, v1, v3
+// GFX12: v_dual_sub_f32 v255, v4, v2 :: v_dual_add_f32 v6, v1, v3 ; encoding: [0x04,0x05,0x48,0xc9,0x01,0x07,0x06,0xff]
+// W64-ERR: :[[@LINE-2]]:{{[0-9]+}}: error
+
+v_dual_sub_f32 v255, v1, v2 :: v_dual_add_f32 v6, v255, v3
+// GFX12: v_dual_sub_f32 v255, v1, v2 :: v_dual_add_f32 v6, v255, v3 ; encoding: [0x01,0x05,0x48,0xc9,0xff,0x07,0x06,0xff]
+// W64-ERR: :[[@LINE-2]]:{{[0-9]+}}: error
```

Comment:
Missing check for what the error is?

Reply:
That is 'error: instruction requires wavesize=32' but I will probably have to change the test checks to autogenerated after it is merged down.

Reply:
I.e.: https://github.com/llvm/llvm-project/pull/147918

This one is once again too large to render in github, but essentially changes checks to what update_mc_test_checks.py did with the full error message. I may push it after parents successfully land and merge.